helm/msgsvc/charts/activemq/values.yaml

```diff
@@ -47,10 +53,14 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+initContainers:
+  enabled: true
```

Comment:
Set to false since this is CERN specific. The same comment for fsGroup: 101 below.

Reply:
updated.